salt/_modules/metalk8s_service_configuration.py

```diff
@@ -27,6 +29,7 @@ def get_service_conf(
     Arguments:
         configmap_name: the ConfigMap name
         namespace: the Namespace where the ConfigMap is stored
+        default_csc: the csc configurations imported from YAML file
```

Comment:
Should be `CSC`, not `csc`. And isn't `csc configurations` redundant as it stands for `Cluster & Services Configurations` ?

Reply:
Let's go with `CSC`
Done